torch_xla/csrc/tensor_methods.cpp

```diff
@@ -2089,12 +2089,20 @@ XLATensorPtr pow(const XLATensorPtr& input, const at::Scalar& exponent) {
   // We want to pass exponent_node as a constant to give XLA more room to
   // optimize
   torch::lazy::Value exponent_node =
-      XLAGraphExecutor::Get()->GetIrValueForConstant(exponent, input->shape());
-  return input->CreateFrom(Pow(input->GetIrValue(), exponent_node));
+      XLAGraphExecutor::Get()->GetIrValueForConstant(exponent);
+  torch::lazy::NodePtr node = Pow(input->GetIrValue(), exponent_node);
+  auto* xla_node = dynamic_cast<XlaNode*>(node.get());
+  at::ScalarType dtype =
+      TorchTypeFromXlaType(xla_node->xla_shape().element_type());
+  return input->CreateFrom(node, dtype);
```

Comment:
I think you can do something similar to `input->CreateFrom(node, /*logical_element_type=*/nullptr)` to make sure `logical_lement_type` is not being inherited from input, and it will by defualt use the `xla_shape`.

Reply:
done.